101-bundle-json.md

```diff
@@ -573,5 +602,35 @@ The fields are defined as follows:
 
 The usage of extensions is undefined. However, bundles SHOULD be installable by runtimes that do not understand the extensions.
 
+## Outputs
+
+The `outputs` section of the `bundle.json` defines which outputs an application will produce during the course of executing a bundle. Outputs are expected to be written to one or more files on the file system of the invocation image. The location of this file MUST be provided in the output definition.
+
+Output specifications are flat (not tree-like), consisting of name/value pairs. The output definition includes a destination the output will be written to, along with a type to help bundle users identify how to consume them.
+
+```json
+"outputs" : {
+    "hostName" : {
+        "type" : "string",
+        "path" : "/cnab/app/outputs/hostname"
+    },
+    "port" : {
+        "type" : "integer",
+        "path" : "/cnab/app/outputs/port"
+    },
+    "clientCert" : {
+        "type" : "file",
+        "path" : "/cnab/app/outputs/clientCert"
+    }
+}
+```
+
+- `output`: name/value pairs describing an application output
+  - `<name>`: The name of the output. This is REQUIRED. In the example above: `hostName`, `port`, `clientCert`. This
+    is mapped to a value definition, which contains the following fields:
+    - `type`: "null", "boolean", "string", "number", or "integer" or "file". These types correspond to the main primitive types of JSON (with _integer_ and _file_ as a special case). (REQUIRED)
```

Comment:
Perhaps link / more detailed explanation of the distinction between “integer” and “number”?